colour/temperature/ohno2013.py

```diff
@@ -80,16 +83,14 @@ def planckian_table(
 
     Parameters
     ----------
-    uv
-        *uv* chromaticity coordinates.
     cmfs
         Standard observer colour matching functions.
     start
         Temperature range start in kelvin degrees.
     end
         Temperature range end in kelvin degrees.
-    count
-        Temperatures count in the planckian table.
+    spacing
+        The spacing between values expressed as a multiplier, default = 1.01
```

Comment:
**Nice/Minor**

Quite the radical change!

The reported default does not seem to be correct.